### DISCUSSION TITLE
Philips hue Infuse lamp

### DISCUSSION THREAD

**Discussion by emilowich** [Koenkk/zigbee2mqtt#11364] [Devices]
I am completely new to zigbee2mqtt. Changed over from phoscon as I have heard many good things. I have moved over most of my lights, sensors and so on, working well. 
Today however I got my new Philips hue infuse lamp mounted and to my surprise zigbee2mqtt says the device is unsupported, as far as I can tell the current version 1.23.0 should add support for this lamp, however device is listed in the change log as 915005997501, but my lamp is showing up as 915005997301. 
Being completely new to zigbee2mqtt, how can I easily get my lamp to work as intended?

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of this device?

**Reply by emilowich** [reply]
Maybe, if you can help me as to where to find it?

**Reply by Koenkk** [reply]
You can find it in the z2m data directory, it is next to the `configuration.yaml`

**Reply by emilowich** [reply]
Okay, thanks, I found it. 
This is the line for the Philips hue lamp:
{"id":43,"type":"Router","ieeeAddr":"0x0017880109c02ce2","nwkAddr":11786,"manufId":4107,"manufName":"Philips","powerSource":"Mains (single phase)","modelId":"[915005997301](tel:915005997301)","epList":[11,242],"endpoints":{"11":{"profId":260,"epId":11,"devId":269,"inClusterList":[0,3,4,5,6,8,4096,768,64513],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"[915005997301](tel:915005997301)","manufacturerName":"Philips","powerSource":1,"zclVersion":2,"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210305","swBuildId":"1.82.10"}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210305","swBuildId":"1.82.10","zclVersion":2,"interviewCompleted":true,"meta":{},"lastSeen":1644656225597,"defaultSendRequestWhen":"immediate"}

**Reply by Koenkk** [reply]
added.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by emilowich** [reply]
Super, thank you. I will give it a go later.
